<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: transparent;
	background: transparent;
	padding: 0;
	margin: 0;
	color: #fff;
}
#centralwidget{
	background-color: #1f232a;
}
#leftMenuSubContainer{
	background-color: #16191d;
}
#leftMenuSubContainer QPushButton{
	text-align: left;
	padding: 5px 10px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}
#centerMenuSubContainer, #rightMenuSubContainer{
	background-color: #2c313c;
}
#frame_4, #frame_8, #popupNotificationSubContainer{
	background-color: #16191d;
	border-radius: 2px;
}
#headerContainer, #footerContainter{
	background-color: #2c313c;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="leftMenuContainer" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="leftMenuSubContainer" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="menuBtn">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/align-justify.png</normaloff>:/icons/Icons/align-justify.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="honBtn">
               <property name="styleSheet">
                <string notr="true">background-color:#1f232a;</string>
               </property>
               <property name="text">
                <string>홈</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/home.png</normaloff>:/icons/Icons/home.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startBtn">
               <property name="text">
                <string>실행하기</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/play.png</normaloff>:/icons/Icons/play.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="attatchfile1">
               <property name="text">
                <string>파일첨부</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/file-plus.png</normaloff>:/icons/Icons/file-plus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="attatchfile2">
               <property name="text">
                <string>파일첨부</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/file-plus.png</normaloff>:/icons/Icons/file-plus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="attatchfile3">
               <property name="text">
                <string>파일첨부</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/file-plus.png</normaloff>:/icons/Icons/file-plus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveBtn">
               <property name="text">
                <string>저장</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/save.png</normaloff>:/icons/Icons/save.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="informationBtn">
               <property name="text">
                <string>정보</string>
               </property>
               <property name="icon">
                <iconset resource="QSS/QSS_Resources.qrc">
                 <normaloff>:/icons/Icons/info.png</normaloff>:/icons/Icons/info.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainbodycontainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>671</width>
         <height>71</height>
        </rect>
       </property>
       <widget class="QLabel" name="label2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>521</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>파일첨부2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>521</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>파일첨부3</string>
        </property>
       </widget>
       <widget class="QLabel" name="label1">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>521</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>파일첨부1</string>
        </property>
       </widget>
       <widget class="QLabel" name="title">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>48</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>매트릭스</string>
        </property>
       </widget>
       <widget class="QLabel" name="title2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>48</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>기점</string>
        </property>
       </widget>
       <widget class="QLabel" name="title3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>48</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>종점</string>
        </property>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>1031</width>
         <height>231</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>1031</width>
         <height>251</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>340</y>
         <width>751</width>
         <height>21</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="QSS/QSS_Resources.qrc"/>
  <include location="QSS/resource.qrc"/>
 </resources>
 <connections/>
</ui>
